lib/presentation/notifiers/active_room_id_notifier.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:fluffychat/utils/extension/get_it_extension.dart';
+import 'package:flutter/widgets.dart';
+
+class ActiveRoomIdNotifier extends ValueNotifier<String?> {
```

Comment:
why do you need to have `ActiveRoomIdNotifier`

Reply:
to be able to update it inside `onContactTap`  (from `go_to_direct_chat_mixin` ) and synchronize the chat list so the right chat is highlighted 

https://github.com/linagora/twake-on-matrix/blob/abe5f18f87eb7d7064fc9a77ab38b984d96f9cc2/lib/presentation/mixins/go_to_direct_chat_mixin.dart#L47

https://github.com/linagora/twake-on-matrix/pull/1596#issuecomment-2019499734


see also this comment for more informations: https://github.com/linagora/twake-on-matrix/pull/1596#discussion_r1540622496

Reply:
https://github.com/linagora/twake-on-matrix/pull/1596#discussion_r1540611424